File: src/app/src/main/java/com/huawei/training/java/utils/network/NetworkConnectionCallbacks.java
package com.huawei.training.java.utils.network;


/**
 * @since 2020
 * @author Huawei DTSE India
 */
public interface NetworkConnectionCallbacks {
    /**
     * On net connected.
     */
    void onNetConnected();

    /**
     * On net disconnected.
     */
    void onNetDisconnected();
}

File: src/app/src/main/java/com/huawei/training/kotlin/utils/network/NetworkConnectionCallbacks.kt
package com.huawei.training.kotlin.utils.network

/**
 * @since 2020
 * @author Huawei DTSE India
 */
interface NetworkConnectionCallbacks {
    /**
     * On net connected.
     */
    fun onNetConnected()

    /**
     * On net disconnected.
     */
    fun onNetDisconnected()
}